### PR TITLE
add whats new for astropy.stats

### PR DESCRIPTION
**Pull request by mirca**
cc @crawfordsm

### CODE REVIEW

**Comment by pllim** [comment]
What about Lomb-Scargle?

**Comment by astrofrog** [comment]
@pllim - there is a separate section for that

File: docs/whatsnew/1.2.rst [hunk 1]
```diff
@@ -43,7 +43,11 @@ Lomb-scargle periodograms
 
 Other new statistics features
 -----------------------------
-
+* The Jackknife resampling method is available in `astropy.stats.jackknife`.
+* Circular statistics (circular mean, variance, etc) are provided in
+  `astropy.stats.circstats`.
```
Comment (crawfordsm) [inline docs/whatsnew/1.2.rst]: Add:  

The circular stats are compatible with `astropy.unit.Quantity` and follow the same conventions as used in R.

**Comment by crawfordsm** [comment]
Thanks for the update @mirca -- can you add similar descriptions for Jackknife and the info theory so that people can know what they can be used for and what they can be done?

**Comment by astrofrog** [comment]
Thanks @mirca!

**Comment by astrofrog** [comment]
👻

**Pull request merged**